### PR TITLE
[move] Fix README

### PR DESCRIPTION
**Pull request by tnowacki**
## Description 

- Fix outdated readme
- Remove no longer updated `changes` directory
- Fix stranded test

## Test Plan 

👀 

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by dariorussi — APPROVED** [review]
this looks really good to me. THe README is good, I downloaded the PR and tried all the links and they all work great.
Let's ship it?

File: external-crates/move/README.md [hunk 1]
```diff
@@ -4,21 +4,24 @@ title: Move Language
...
 ## Overview
 
-The Move language directory consists of four main parts:
-
-- [virtual machine](move-vm/) (VM) &mdash; contains the bytecode format, a bytecode interpreter, and infrastructure for executing a block of transactions. This directory also contains the infrastructure to generate the genesis block.
-
-- [bytecode verifier](move-bytecode-verifier/) &mdash; contains a static analysis tool for rejecting invalid Move bytecode. The virtual machine runs the bytecode verifier on any new Move code it encounters before executing it. The compiler runs the bytecode verifier on its output and surfaces the errors to the programmer.
-
-- [move-compiler](move-compiler/) &mdash; contains the Move source language compiler.
+The Move language directory consists of three distinct parts: the Move bytecode language, the Move IR, and the Move source language.
```
Comment (ronny-mysten) [inline external-crates/move/README.md:11]:
```suggestion
The Move language directory consists of three distinct parts: the Move bytecode language, the Move intermediate representation (IR), and the Move source language.
```
Suggest this on first use for those folks that might not immediately know what IR stands for and have to use Google or AI to be sure. I'm not one of those people, of course, but there might be some out there...

File: external-crates/move/README.md [hunk 2]
```diff
@@ -4,21 +4,24 @@ title: Move Language
...
 
-- [standard library](move-stdlib/) &mdash; contains the standard library transaction scripts.
+- The Move bytecode language defines programs published to the blockchain. It has a static type system that guarantees the absence of certain critical errors, including the prevention of duplicating certain values. Relevant crates include:
+  - [move-binary-format](crates/move-binary-format/) defines the binary format for Move bytecode.
+  - [move-bytecode-verifier](crates/move-bytecode-verifier/) provides the static safety checks for Move bytecode.
+  - [move-vm-types](crates/move-vm-types/) provides a shared utility crate types used by the Move VM and adapting layers.
+  - [move-vm-runtime](crates/move-vm-runtime/) provides the runtime for the Move VM, used for executing Move programs.
 
-## Exploring the Move language
+- The Move IR is a very low-level intermediate representation that closely mirrors Move bytecode. What mainly differentiates it from the bytecode is that names are used as opposed to indexes into pools/tables. Relevant creates include:
```
Comment (ronny-mysten) [inline external-crates/move/README.md:19]:
```suggestion
- The Move IR is a low-level intermediate representation that closely mirrors Move bytecode. What mainly differentiates it from the bytecode is that names are used as opposed to indexes into pools/tables. Relevant crates include:
```

File: external-crates/move/README.md [hunk 3]
```diff
@@ -4,21 +4,24 @@ title: Move Language
...
+  - [move-ir-types](crates/move-ir-types/) defines the IRs Rust types, notably the AST.
+  - [move-ir](crates/move-ir-to-bytecode/) compiles the IR to the bytecode.
 
-- You can find many small Move examples in the [tests](move-compiler/tests/move_check/) directory. The easiest way to experiment with Move is to create a new test in this directory and run it with `cargo test`.
+- The Move source language is a high-level language that compiles to Move bytecode. It is designed to be a familiar and ergonomic language for developers that provides minimal abstractions over the Move bytecode. Relevant creates include:
```
Comment (ronny-mysten) [inline external-crates/move/README.md:23]:
```suggestion
- The Move source language is a high-level language that compiles to Move bytecode. It is designed to be a familiar and ergonomic language for developers that provides minimal abstractions over the Move bytecode. Relevant crates include:
```

File: external-crates/move/README.md [hunk 4]
```diff
@@ -4,21 +4,24 @@ title: Move Language
...
+  - [move-compiler](crates/move-compiler/) defines the source language and all compilation steps up to the Move IR
```
Comment (ronny-mysten) [inline external-crates/move/README.md:24]:
```suggestion
  - [move-compiler](crates/move-compiler/) defines the source language and all compilation steps up to the Move IR.
```

File: external-crates/move/README.md [hunk 5]
```diff
@@ -4,21 +4,24 @@ title: Move Language
...
+  - [move-analyzer](crates/move-analyzer/) provides the IDE integration for the source language
```
Comment (ronny-mysten) [inline external-crates/move/README.md:25]:
```suggestion
  - [move-analyzer](crates/move-analyzer/) provides the IDE integration for the source language.
```

**Review by ronny-mysten — COMMENTED** [review]
Pickin nits

**Pull request merged**